openid-4-verifiable-presentations-1_0.md

```diff
@@ -1361,21 +1361,21 @@ This specification defines how the Verifier can determine Credential formats, pr
 
 This specification defines new metadata parameters according to [@!RFC8414].
 
-* `vp_formats_supported`: REQUIRED. An object containing a list of name/value pairs, where the name is a string identifying a Credential format supported by the Wallet. Valid Credential format identifier values are defined in (#format_specific_parameters). Other values may be used when defined in the profiles of this specification. The value is an object containing a parameter defined below:
-    * `alg_values_supported`: OPTIONAL. An object where the value is an array of case sensitive strings that identify the cryptographic suites that are supported. Parties will need to agree upon the meanings of the values used, which may be context-specific. For specific values that can be used depending on the Credential format, see (#format_specific_parameters). If `alg_values_supported` is omitted, it is unknown what cryptographic suites the wallet supports.
+* `vp_formats_supported`: REQUIRED. An object defining the formats and proof types of Verifiable Presentations and Verifiable Credentials that a Wallet supports. For specific values that can be used, see (#format_specific_parameters).
```

Comment:
Why is the Wallet metadata called  `vp_formats_supported` and Verifier metadata just `vp_formats`?

Reply:
I'm not sure, it was already like this. Do you propose we make them the same? Should it both be `vp_formats_supported` (since i removed the `supported` from the properties (e.g. `alg_values_supported` became `alg_values`

Reply:
I don't mind too much what the name is, but I think it should be the same for wallet and verifier, because it has the same values, right? In OpenID4VCI many issuer metadata claim names have `_supported` keys.

Reply:
@paulbastian this is the naming convention we inherited from OIDC... @selfissued